backend/flightData.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from datetime import datetime
+
+def get_flight_data(aircraft: str, distance: int):
+    url = f"https://despouy.ca/flight-fuel-api/q/?aircraft={aircraft}&distance={distance}"
```

Comment:
did we double check this from the source????

Reply:
I tested against Postman request for the same parameters and got the same and expected response.